### PR TITLE
Add support for Chinese

### PR DESCRIPTION
**Pull request by AirsQ**
Hi, here is the new version supprot for Chinese PR.
The new version code fixed by hguy.
He helped a lot, thanks!

Screenshot：

### CODE REVIEW

File: src/TQVaultAE.Data/Database.cs [hunk 1]
```diff
@@ -899,7 +899,7 @@ private void ParseTextDB(string databaseFile, string filename)
 					if (Regex.Match(label, @"^(?<Tag>\[\w+\])(?<Label>[^\\[]+)|^\[(?<Label>[^\]]+)\]$") is { Success: true } match)
 						label = match.Groups["Label"].Value.Trim();
 					else
```
[the remark this answers is not in the document]
Comment (AirsQ) [reply src/TQVaultAE.Data/Database.cs:901]: oh, you are right.

[the remark this answers is not in the document]
Comment (AirsQ) [reply src/TQVaultAE.Data/Database.cs]: Did you find any other problems? I handle it together.

[the remark this answers is not in the document]
Comment (hguy) [reply src/TQVaultAE.Data/Database.cs]: no it's fine

[the remark this answers is not in the document]
Comment (AirsQ) [reply src/TQVaultAE.Data/Database.cs]: opps, I just realized that PR will auto include new commit.
I shouldn't have closed the last PR. 
 w(ﾟДﾟ)w

Now, can only maintain the status quo.  o(〃'▽'〃)o

**Comment by EtienneLamoureux** [comment]
@hguy Would you might looking this over and letting me know if any of it is problematic?

**Comment by hguy** [comment]
> @hguy Would you might looking this over and letting me know if any of it is problematic?

@EtienneLamoureux As far as i know it was OK and ready to merge

**Pull request merged**